### ISSUE TITLE
Modify the PDF Designs to align with Developed PDF

### ISSUE THREAD

**Issue by lbeatonn** [hackforla/tdm-calculator#1761]
### Overview

We need to modify the PDF to align more closely with what was developed and also it needs to adhere to the design system.

### Action Items
 
- [x] Ensure that the blue color used is in the design system
- [x] Remove the grey background and increase the weight of the text on the grey background to bold (grey background is not feasible for dev) 

### Resources:

[FIGMA LINK](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=13534-12225&t=EqKs0KQErs7uxDfJ-1)

![Image](https://github.com/hackforla/tdm-calculator/assets/91226916/36b4dcd5-17ea-43aa-8b3c-dbc2498dcfb8)

**Comment by ancyjoy7** [comment]
![Summary Page PDF matched to dev version](https://github.com/user-attachments/assets/270f71e4-a7f7-42af-a2c1-26728325ac88)

This is the first iteration where the design was matched to the dev teams version of the summary page pdf.

![Summary Page PDF - latest version](https://github.com/user-attachments/assets/a53155fd-5c10-4d7f-9654-6fb9f4946b5c)

![Summary Page PDF - lates version B W](https://github.com/user-attachments/assets/a2f72ec9-58f8-4d66-b952-41dd7487c961)

Over the break, I worked on the new iterations with the goal of better readability. Fixed the alignment and added horizontal lines to headers.

What's different in the latest iteration?

- Addition of horizontal lines under the headers.
- Changed the headers' color from Cyan Blue (#43739D) to Dark Blue (#00122B) for better readability when the pdf is printed in black and white.
- Updated the alignment and spacing between elements.
